searx/settings.yml

```diff
@@ -1599,6 +1599,12 @@ engines:
       website: https://www.wikivoyage.org/
       wikidata_id: Q373
 
+  - name: wikicommons.images
+    engine: wikicommons
+    shortcut: wc
+    categories: images
+    number_of_results: 10
```

Comment:
I think we should comment that out by default, or completely remove it?

Reply:
Some other engines specify the number_of_results.
I'm inclined to keep this setting without much conviction.